### DISCUSSION TITLE
What happens when you click thumbs up or down?

### DISCUSSION THREAD

**Discussion by tdiprima** [open-webui/open-webui#6516] [General]
I found out from another issue (below) that the data doesn't get sent to anybody.  (And it doesn't - I checked the Network tab of the browser.)  But what I want to know is &mdash; what _actually_ happens?  Does the data help refine response styles or prioritize certain types of answers based on the feedback?  Or does the data get stored somewhere and it's up to me to fetch it and do something with it?

_Ref:_ #225

**Comment by tensiondriven** [comment]
I think this is used for the model comparison scores which are calclulated/stored/displayed locally.

**Comment by lleevveell66** [comment]
Googled, wondering the same.   Brought me here.  Thanks for asking/answering.
